docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -709,7 +537,7 @@ However, an object may have multiple rules for a single role. For example:
 
```

Comment:
What is an object, a rule and a role here?

Reply:
Changing to `However, an object (database, table, partition) `..